homeassistant/components/livisi/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "LIVISI Smart Home",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/livisi",
-  "requirements": ["aiolivisi==0.0.14"],
+  "requirements": ["aiolivisi==0.0.15"],
```

Comment:
How is the dependency bump linked to reauth?
There are no release notes and no tags in the library, so what are the changes between 0.0.14 and 0.0.15?

Reply:
In the 0.0.15 version, aiolivisi raises the TokenExpiredException when the authentication token is expired.